seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpGetCommandProcessor.java

```diff
@@ -159,6 +166,19 @@ private void getSystemMonitoringInformation(HttpGetCommand command) {
         this.prepareResponse(command, jsonValues);
     }
 
+    private void handleLog(HttpGetCommand command) throws IOException {
+
+        String logPath = System.getProperty("seatunnel.logs.path");
+        String logName = System.getProperty("seatunnel.logs.file_name") + ".log";
```

Comment:
Why not use config in ${SEATUNNEL_HOME}/config/log4j2.properties to get file path? Declaring the log path with different keys in different places will increase maintenance costs.

Reply:
> Why not use config in ${SEATUNNEL_HOME}/config/log4j2.properties to get file path? Declaring the log path with different keys in different places will increase maintenance costs. 

How to get this? Can you elaborate?

Reply:
oh, seem like we already introduce `seatunnel.logs.path` and `seatunnel.logs.file_name` environment variables before this PR. Ignore this one.